content/basics/ably.textile

```diff
@@ -0,0 +1,92 @@
+
+* "Pub/Sub with channels":#pubsub
+* "Presence and occupancy":#presence-and-occupancy
+* "Message storage and history":#message-storage-and-history
+* "Push Notifications":#push-notifications
+
+h3(#pubsub). Pub/Sub with channels
+
+Ably uses the Publish/Subscribe (Pub/Sub) architectural pattern to allow any number of publishers to publish data and any number of subscribers to subscribe to a source of data. This works with Ably's concept of channels. Channels are used to aggregate all data into a single location, allowing publishers to publish to just one location and subscribers to subscribe to just one source.
+
+Read more in our "Pub/Sub section":/channels.
+
+h3(#presence-and-occupancy). Presence and occupancy
+
+Ably’s presence feature allows clients or devices to announce their presence on a channel. Other devices or services may then subscribe to these presence events, such as entering, updating their state, or leaving the channel. You can also request a list of clients or devices that are online/offline on a channel at a particular point in time via the REST API.
+
+Occupancy provides metrics about the clients attached to a channel, including a count of the number of clients attached, the number of publishers and subscribers, the number of presence members, and the number of presence publishers and subscribers.
+
+Read more in our "Presence and occupancy section":/presence-occupancy.
+
+h3(#message-storage-and-history). Message storage and history
+
+By default messages are only stored for two minutes on Ably’s servers, these messages are retrieved automatically by the SDK in the event of a lost connection. However, if you've enabled message persistence and the disconnection is longer than two minutes you can use the History API to retrieve messages.
+
+Ably's message storage and history allows messages to be accessed after initially being sent. You can persist the last message sent using the rewind channel option or enable persistence of all messages for 24 hours for the free tier or 72 hours for paid tiers. These messages can be retrieved using the History API. You can also persist messages to external storage services such as AWS through the use of "Ably Integrations":/integrations.
+
+Read more in our "Message storage and history section":/storage-history.
+
+h3(#push-notifications). Push notifications
+
+Ably's Push Notifications feature allows messages to be pushed to subscribers without needing a device to maintain a connection to Ably. Push Notifications are commonly used to display visual notifications to users or launch background processes for apps in a battery-efficient manner.
+
+Read more in our "Push Notifications section":/push.
+
+h2(#infrastructure). About the infrastructure
+
+To ensure unrivaled realtime messaging and data streaming performance, availability, and reliability, the Ably Global Network has 7 geographically-distributed core routing datacenters and 307 edge acceleration points-of-presence(PoPs).
```

Comment:
There's a lot going on in this sentence. I'd rewrite to: 
Ably has a Global Network of 7 geographically-distributed core routing data centers and 307 edge acceleration points-of-presence(PoPs). Ably's Global Network ensures unrivaled realtime messaging, data streaming performance, availability, and reliability.